css.md

```diff
@@ -209,24 +228,49 @@ $heading-font: 200 16px/1.3 $title-font-family
 // Defaults
 $default-radius: 6px
 $default-duration: .25s
+$default-vertical-spacing: ($gutter * 2)
 ```
 
-### Application (`application.css.sass`)
+### Application (`application.sass`)
 
-This should ideally be the only file that is included directly via `stylesheet_link_tag`. We also want to define specific order of import for the `lib` files as we may be making use of `_functions.css.sass` in `_variables.css.sass` or `_variables.css.sass` in `_mixins.css.sass`.
+This should ideally be the only file that is included directly via `stylesheet_link_tag`. We also want to define specific order of import for the `lib` files as we may be making use of `_functions.sass` in `_variables.sass` or `_variables.sass` in `_mixins.sass`.
 
-> Note that SASS `@import` does not require the explicit use of underscores for filenames. Also be sure to use `@import "folder/**/*"` to import all files and folders under a particular directory.
+ > *Note:* SASS `@import` does not require the explicit use of underscores for filenames. Also be sure to use `@import "folder/**/*"` to import all files and folders under a particular directory.
 
 ```sass
-@import "compass", "compass/reset"
-
+// To "reset" browser-styles we either use
+@import "normalize-rails"
+// or
+@import "compass"
+@import "compass/reset"
+
+/*!
+ * Author:       The Frontier Group
+ * Author URI:   http://www.thefrontiergroup.com.au/
+ */
```

Comment:
These days most of the plugins we use have their own banners at the top.

the `/*! */` makes it a permanent comment (meaning, it will not get deleted by minification).

Not much we can do if they don't have one :/ Unless we add another generic banner for "vendors"